src/types/iris.ts

```diff
@@ -0,0 +1,46 @@
+// IRIS API Type Definitions
+
+export interface User {
+  id: number;
+  name: string;
+  created: string;
+  admin: boolean;
```

Comment:
Prefixing `boolean` members with `is`/`are` makes it more clear that they are `boolean` values.